### PR TITLE
Assignment 2 Angel Lopez Monzoncillo

### PR DESCRIPTION
**Pull request by angellopezmonzoncillo**

### CODE REVIEW

File: Assignment2/angellopezmonzoncillo.rdf [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0">
+	<rdf: RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-sintax-ns#"
+			  xmlns:person="http://...../person#"
```
Comment (ocorcho) [inline Assignment2/angellopezmonzoncillo.rdf]: The ... should be revised, of course.

File: Assignment2/angellopezmonzoncillo.ttl [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+@prefix person:<http://....../person#>
+@prefix sensor:<http://....../sensor#>
+@prefix computer:<http://....../computer#>
+@prefix measurement:<http://....../measurement#>
+		 
+:class01 class: include: sensor029;
+		 class: include: computer101;
+:sensor029 sensor: hasMeasurement: measurement8401;
+:computer101 computer: hasOwner: user10A;
+:measurement8401 measurement: hasTemperature: "29";
```
Comment (ocorcho) [inline Assignment2/angellopezmonzoncillo.ttl]: Check datatypes here and in the following line.

**Comment by ocorcho** [comment]
Are you going to review the changes that I proposed?

**Comment by angellopezmonzoncillo** [comment]
Yes.

**Comment by angellopezmonzoncillo** [comment]
Updated with proposed changes

File: Assignment2/angellopezmonzoncillo.ttl (hunk 2 above)
Comment (ocorcho) [reply Assignment2/angellopezmonzoncillo.ttl]: No changes have been done, as I originally suggested.

**Comment by angellopezmonzoncillo** [comment]
I sent a pull request, but maybe i did It wrong, later i will try to do It
well

**Comment by angellopezmonzoncillo** [comment]
I press pull request but does not appear the create pull request option in the subsequent window

File: Assignment2/angellopezmonzoncillo.ttl [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+@prefix person:<http://www.ontologies.org/ontologies/person#>
+@prefix sensor:<http://www.ontologies.org/ontologies/sensor#>
+@prefix computer:<http://www.ontologies.org/ontologies/computer#>
+@prefix measurement:<http://www.ontologies.org/ontologies/measurement#>
+		 
+:class01 class: include: sensor029;
+		 class: include: computer101;
+:sensor029 sensor: hasMeasurement: measurement8401;
+:computer101 computer: hasOwner: user10A;
+:measurement8401 measurement: hasTemperature: "29";
```
Comment (ocorcho) [inline Assignment2/angellopezmonzoncillo.ttl]: the space between measurement and hasTemperature is wrong. Temperature RDF literal is not correctly typed.

**Comment by angellopezmonzoncillo** [comment]
I did not see the proposed changes about the space between measurement and
as temperature, sorry

**Pull request closed without merging**